docker-compose.yml

```diff
@@ -25,11 +25,15 @@ services:
       MYSQL_ROOT_PASSWORD: password
 
   web: &web
+    build:
+      context: .
+      dockerfile: compose/Dockerfile
     env_file:
-      - .env
-    build: .
+      - ./.envs/.local
     image: web
     container_name: web
+    restart: on-failure
+    entrypoint: /wait-for-mysql
     command: python manage.py runserver 0.0.0.0:8000
```

Comment:
Bør ikke entrypoint peke mot `/compose/wait-for-mysql`?

Hva gjør egt entrypoint her?

Reply:
Den overskriver CMD kommandoen i docker bildet, men ikke kommandoen under her. Blir som å legge inn et steg før command